crosswalks/spase-v2.6.1.csv

```diff
@@ -0,0 +1,74 @@
+Property,SPASE
+codeRepository,RepositoryID
+programmingLanguage,codeLanguage
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,AccessURL/URL
+fileSize,
+installUrl,ExecutionEnvironment/Installer/URL
+memoryRequirements,ExecutionEnvironment/Memory
+operatingSystem,OperatingSystem
+permissions,
+processorRequirements,ExecutionEnvironment/Cores
```

Comment:
Doesn't seem to match, see https://schema.org/processorRequirements while https://spase-group.org/data/model/spase-2.6.1/index.html has type `xsd:double` (presumably that's a number of cores)

Reply:
You're right, I removed this mapping